### ISSUE TITLE
Broken Link in Example/README.md file?

### ISSUE THREAD

**Issue by mbreakey3** [PowerShell/xWebAdministration#204]
The link to the zip file doesn't appear to be working. What is this [README.md](https://github.com/PowerShell/xWebAdministration/blob/dev/Examples/README.md) file for? There's no clear documentation on it.

**Comment by tysonjhayes** [comment]
Pretty sure that's cruft left over from when we first released.

However the bits they may be referring to may be found here: [DSC-Windows Server 2012 R2-Preview Demos](https://msdnshared.blob.core.windows.net/media/MSDNBlogsFS/prod.evol.blogs.msdn.com/CommunityServer.Blogs.Components.WeblogFiles/00/00/00/63/74/metablogapi/3124.Demo_WindowServer2012R2-Preview_4677B514.zip) and when extracted located under the PreReq\BakeryWebsite path.

That being said. If we *really* want to provide a website for everything to work (not a bad idea) I'm thinking we should have the community redo it so it can be correctly licensed and shipped with the module.

**Comment by rosberglinhares** [comment]
The license in [this](https://msdn.microsoft.com/en-us/cc300389) link says:

> **11. Definitions.**
> “Services” means the Developer Services, Documentation Portals, the http://connect.microsoft.com site, and Microsoft Software we make available to you under this Agreement.
> “Documentation Portals” means the Microsoft developer network content and marketing site available at http://msdn.microsoft.com" and information technology specialist content and marketing site available at http://technet.microsoft.com, or at alternate sites we identify.
> “Microsoft Software” means Microsoft software and computer code, including sample code and Developer Services Software.
> 
> **1.a. Right to use.** We grant you the right to access and use the Services in accordance with this Agreement.
> 
> **2.a. Using Microsoft Software outside the Service.** [...] Microsoft may provide you with Microsoft Software for use outside the Developer Services and with (1) the Developer Services or (2) programs you develop using the Developer Services. If the Microsoft Software is provided with its own license terms, those terms control as modified by the foregoing. If the Microsoft Software does not have its own license terms, then you may install and use any number of copies of the Microsoft Software to design, develop, and test your programs on devices. This subsection does not apply to Microsoft Software addressed in subsection (b) below.
> 
> **2.b. Software on Documentation Portals.** Software accessible on the Documentation Portals is made available by the designated publisher under the associated license terms. If Software is accessible on the Documentation Portals without license terms, then subject subsection (c) below you may use it to design, develop, and test your programs. If any such Software without license terms is marked as “sample” or “example,” then you may use it under the terms of the Microsoft Limited Public License.

What would be the correct way to license the embedded site?

**Comment by johlju** [comment]
I agree that the README.md should be updated and remove the links to the web site demo. Such web site could be built by the integration tests, and the integration tests can build a more complex web site containing of different web sites and web applications (etc.) to show off but also test all the functionality.
